src/Views/LoginPage/components/Form/Form.js

```diff
@@ -1,15 +1,19 @@
-import React from "react";
+import React, { useState } from "react";
 import Input from "../Input/Input";
 import SubmitButton from "../SubmitButton/SubmitButton";
 import "./styles.css";
 import PropTypes from "prop-types";
+import {emailValidation} from "./validation/emailValidation";
```

Comment:
Task: 
"1. Create file 'validation' inside LoginPage dir. 
2. Create function 'emailValidation' ". 
Instead, you create folder 'validation' and file 'emailValidation'. Please, change it

Reply:
changed